frontend/src/util/window.ts

```diff
@@ -1,42 +1,16 @@
-import { Admin } from '@graph/schemas'
-import { INTERCOM_APP_ID } from '@util/constants/constants'
-import { H } from 'highlight.run'
+import { message } from 'antd'
 
 import { FRONTEND_URI } from '@/constants'
 
 export function GetBaseURL(): string {
 	return FRONTEND_URI
 }
 
-interface IntercomInit {
-	admin?: Admin
+export const showSupportBubble = () => {
+	window.open('https://highlight.io/community', '_blank')
 }
 
-export function loadIntercom({ admin }: IntercomInit = {}) {
```

Comment:
Looks like we are still trying to import this function from a couple files
- frontend/src/index.tsx
- frontend/src/routers/AppRouter/AppRouter.tsx

Reply:
whoops, good catch!